### ISSUE TITLE
"mbed publish" sends to a 404 in case the project is not Git-enabled

### ISSUE THREAD

**Issue by kartben** [ARMmbed/mbed-cli#362]
Steps to reproduce:

```
mbed new test
cd test
mbed publish
```

This outputs the following message:

```
[mbed] Checking for local modifications...
[mbed] ERROR: Program "test" in "/private/tmp/test" is a local repository.
[mbed] ERROR: Please associate it with a remote repository URL before attempting to publish.
[mbed] ERROR: Read more about git repositories here:
[mbed] ERROR: http://developer.mbed.org/handbook/how-to-publish-with-git/
```

http://developer.mbed.org/handbook/how-to-publish-with-git/ is a 404.

**Comment by screamerbg** [comment]
Fixed in https://github.com/ARMmbed/mbed-cli/pull/379

**Comment by screamerbg** [comment]
Now merged and part of the mbed CLI 0.9.10 release. Please reopen if the issue persists.
